external/storm-hdfs/src/test/java/org/apache/storm/hdfs/bolt/TestHdfsBolt.java

```diff
@@ -203,6 +204,17 @@ public void testTickTuples() throws IOException {
         //Tick should have flushed it
         Assert.assertEquals(1, countNonZeroLengthFiles(testRoot));
     }
+    
+    @Test
+    public void testShutdownStorm3372() {
+        //STORM-3372: Rotation policy other than TimedRotationPolicy causes NPE on cleanup
```

Comment:
The test for the issue

Reply:
I prefer to not use jira number as a test case name. Just name the test case as what it is about will be better

Reply:
Makes sense, changed it.